.changeset/tame-frogs-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-yoga/plugin-response-cache': minor
+---
+
+Use ETag to use HTTP caching
```

Comment:
Maybe elaborate a bit more here and also mention `if-none-match`.

This should also be reflected in the docs imho

Reply:
Added details to the docs, and I put a link in the changeset